tests/main/selinux-clean/task.yaml

```diff
@@ -0,0 +1,54 @@
+summary: Check that basic snap management does not raise any SELinux denials
+
+description: |
+    On systems where SELinux is supported, make sure that starting snapd and
+    performing basic install/remove tasks does not cause SELinux denials. Even
+    though we do not support SELinux for enforcing confinement of snaps, we do
+    not want to cause unnecessary warnings when users are performing basic
+    management tasks on snaps.
+
+systems: [fedora-*, centos-*]
+prepare: |
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB"/pkgdb.sh
+
+    # Install some fonts so that the fc-cache helpers have something to work with
+    distro_install_package fontconfig dejavu-sans-fonts
+
+    # Enable enforcing mode, our policy is already marked as permissive, so we
+    # will get audit entries but the program will not be stopped by SELinux
+    setenforce 1
+    ausearch --checkpoint stamp -m AVC || true
+
+restore: |
+    rm -f stamp
+    setenforce 0
```

Comment:
Nitpick: I'd be cleaner if this test had restored the state of selinux rather than disabling it and assuming the initial state was the same.